documentation/core/Modifiers.md

```diff
@@ -0,0 +1,20 @@
+# Modifiers
+A modifier is: ...
+
+The main modifiers are `Attribute` and `Format`.
+
+## Attributes
+When parsing and rendering a document, we need to save the attributes that a
+DOM node has and save it in our VNode.
+
+Attributes exists to:
+- retrieve HTML attributes that a DOM node has originally
+- render the HTML attributes that a VNode has
+
+## Format
+The DOM inline nodes (e.g. span, strong, em, b, a, ...) in HTML are difficult to
+manipulate. We therefore we transform them into a Format modifiers that we
+attach to a VNode through the "modifiers" properties.
+
+Format themselve have modifiers. For instance, a HTML strong tag migh have
```

Comment:
"`Format`s may themselves have modifiers. For instance a bold text may be colored red, which would translate into HTML as a strong tag with a style attribute "color: red", and into Jabberwock as (with the help of `Char`, `Attributes` and `Inline` plugins) a group of `Char` nodes with as modifiers: (1) an extension of the `Format` class that renders as a `strong` tag and (2) `Attributes` containing the "color" style set to red."
Too much? :'D